src/Microsoft.Azure.WebJobs.Logging/FunctionInstanceLogItem.cs

```diff
@@ -29,9 +28,13 @@ public class FunctionInstanceLogItem : IFunctionInstanceBaseEntry
         /// <summary>Gets or sets the Function ID of the ancestor function instance.</summary>
         public Guid? ParentId { get; set; }
 
-        /// <summary>Name of this function. This can be used in querying other instances of this function. </summary>
+        /// <summary>Short Name of this function.</summary>
         public string FunctionName { get; set; }
```

Comment:
FunctionId is an escaped normalized form.  
FunctionId may be "host-foo:41bar", whereas functionName may be "foo-BAR".


---
In reply to: [84856180](https://github.com/Azure/azure-webjobs-sdk/pull/884#discussion_r84856180) [](ancestors = 84856180)